python/agents/swe-agent/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore logs
+logs/
+
+# Ignore json files
+*.json
```

Comment:
This filter may be too much as some other code samples may genuinely need to use json files.